src/App.jsx

```diff
@@ -14,33 +15,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState([...goodsFromServer]);
+  const [isReversed, setReversed] = useState(false);
+  const [sortType, setSortType] = useState('');
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const sortAlphabetically = () => {
```

Comment:
try to move the sort function from the component
also, you can use the `reverse` method to simplify logic in this function
and small advice: if you create a new array, you can assign this logic to const variable and simplify login in `setGoods` method🚀

Reply:
> try to move the sort function from the component also, you can use the `reverse` method to simplify logic in this function and small advice: if you create a new array, you can assign this logic to const variable and simplify login in `setGoods` method🚀

Good point, let's create a function to sort and reverse your goods and move it outside the component